packages/cli/src/presenters/detailed/constants.ts

```diff
@@ -1,3 +1,18 @@
 import chalk from "chalk";
 
 export const indenter = chalk.gray("│ ");
+
+export enum colorCodes {
```

Comment:
[Naming] I generally use PascalCase for enums, do you have a preference?

```suggestion
export enum ColorCodes {
```